lib/components/private/touchable/useVirtualTouchable.treat.ts

```diff
@@ -12,8 +12,8 @@ export const virtualTouchable = style({
     minWidth: hitArea,
     height: '100%',
     top: '50%',
-    left: 0,
-    right: 0,
+    left: -10,
+    right: -10,
```

Comment:
Any context here?

Reply:
This is so that the hit area is more-or-less centred over smaller elements (`IconButton` in this case), rather than hanging out over the right side of the element. The tradeoff is that it means the hit area extends horizontally outside the bounds of _all_ virtual touchable elements, but I actually think that's okay.